playbooks/roles/dstat/defaults/main.yml

```diff
@@ -0,0 +1,16 @@
+# -t: time output
+# -c: cpu stats
+# -m: mem stats
+# -s: swap stats
+# -d: disk stats
+# -n: network stats
+# -p: proc stats
+# -r: io stats
+# -g: page stats
+# -i: interrupt stats
+# -l: load stats
+dstat_options: "-tcmdnl"
+# delay is the delay in seconds between each update
+dstat_delay: "5"
+dstat_output_csv: "{{ ansible_env.HOME }}/dstat.csv"
```

Comment:
ansible_facts['user_dir'] is probably better.

Reply:
are you ok with keeping it this way for now? 

BTW I used that so its in line with other places where we use the HOME like this

Reply:
depends if it's the remote user or the local user :)